bcipy/acquisition/demo/demo_lsl_recorder.py

```diff
@@ -0,0 +1,25 @@
+from bcipy.acquisition.protocols.lsl.lsl_recorder import LslRecorder
+import time
+
+SLEEP = 10
+recorder = LslRecorder(path='.')
+
+try:
+    recorder.start()
+    print(f'\nCollecting data for {SLEEP}s... (Interrupt [Ctl-C] to stop)\n')
```

Comment:
Might want to mention here that data will be recorded to the current directory so people can delete it when they are done. I can't remember what happens if the csv file exists.